lib/plottr_components/src/components/project/Book.js

```diff
@@ -114,19 +137,26 @@ const BookConnector = (connector) => {
       if (!book) {
         return (
           <div className={cx('book-container', 'add', { darkmode: ui.darkMode })}>
-            <div className="book add" onClick={this.props.addBook}>
+            {this.renderTemplatePicker()}
+            <div className="book add">
               <div className="front">
                 <div className="cover add">
                   <div className="book-container__add">
-                    <h1>
+                    <h3 onClick={() => this.props.addBook()}>
```

Comment:
I've seen this a couple of times and just wanted to point out that there is a difference in performance between: `onClick={this.props.addBook}` and `onClick={() => this.props.addBook()}`.
The latter creates a new function each time that the component is rendered.
That means that the component that we're passing that lambda to will be redrawn because of a change in props.

Reply:
oh yup, i did know that. Must have missed this one. Thanks